Octokit/Http/Connection.cs

```diff
@@ -719,7 +719,7 @@ static Exception GetExceptionForForbidden(IResponse response)
         {
             string body = response.Body as string ?? "";
 
-            if (body.Contains("rate limit exceeded"))
+            if (body.Contains("rate limit exceeded") || body.Contains("secondary rate limit"))
```

Comment:
I wonder if it might be better throw a different kind of exception for a secondary rate limit. What do you think? I'm just thinking that it would make it easier to handle this specific case as a caller, without doing string matching.

Reply:
I get where you are coming from, makes sense to me :)

Reply:
I’d just define a new exception and raise that. It should be a quick change, and then we can get this approved and merged ✅

Reply:
I've changed it to be it's own exception.